backend/app/views/spree/admin/customer_returns/new.html.erb

```diff
@@ -33,7 +33,7 @@
         </fieldset>
 
         <%= f.field_container :stock_location do %>
-          <%= f.label :stock_location, Spree.t(:stock_location) %>
+          <%= f.label Spree::StockLocation.model_name.human %>
```

Comment:
Well labels seem to be broken currently with select2 so when it actually works "properly" it activates the select2 bug.  I don't know if we want to implement it properly now so it will be broken but hopefully when select2 is patched it works or have it incorrect right now so if/when it's fixed it behaves properly.